WorkflowTesting/Sources/Internal/RenderExpectations.swift

```diff
@@ -48,7 +48,7 @@ extension RenderTester {
 }
 
 extension RenderTester {
-    internal class ExpectedSideEffect<WorkflowType: Workflow> {
+    internal class ExpectedSideEffect<RenderedWorkflowType: Workflow> {
```

Comment:
are these generic parameters even necessary? can they differ from the value from the enclosing scope (RenderTester)?

Reply:
If I removed them then it failed to compile because ExpectedSideEffect didn't have a generic parameter.

Reply:
I was wondering the same thing if they could differ.

Reply:
I was able to remove the type and change the declarations where it was used and it seems to work.  I didn't see anywhere where the WorkflowType was different for the side effect vs RenderTester.  Pushing up changes for review.